languages/python/exercises/concept/conditionals/.docs/introduction.md

```diff
@@ -0,0 +1,44 @@
+## Conditionals
+
+Conditionals are the most common statements used in programming. Conditionals determine the flow of control in a program. In python the `if` statement is used to control the flow of execution.
+
+In basic syntax of an `if` statement is as follows
+
+```python
+x = 10
+y = 5
+# Variant 1
+if x > y:
+    print("x is greater than y")
+
+>>> x is greater than y
+
+# Variant 2
+x = 5
+y = 10
+
+if x > y:
+    print("x is greater than y")
+else:
+    print("y is greater than x")
+
+>>> y is greater than x
+
+# Variant 3
+x = 5
+y = 10
+z = 20
+if x > y:
+    print("x is greater than y and z")
```

Comment:
Not sure if the message printed should be that one.
Afayk at this point, x is only greater that y, and the idea is that you change the vars values and you test the flow.

The same applies for the messages below